wagtail/wagtailadmin/templates/wagtailadmin/base.html

```diff
@@ -37,3 +48,55 @@
         </div>
     </div>
 {% endblock %}
+
+{# Animation easter-egg: #}
+{% block extra_js %}
+    <script>
+        $(function() {
+          var sensitivity = 8; // the amount of times the user must stroke the wagtail to trigger the animation
+
+          var $logoContainer = $('.wagtail-logo-container__desktop');
```

Comment:
I see that as jQuery squashing legitimate errors that could be handled 😛  You could include `if (!logoContainer) return;` or similar on line 59 and the behavior would be clearer. If at some point you do want this on a page that doesn't have jQuery, or you want automated test coverage without jQuery, it bakes in a hassle. The only legitimate use here in my opinion is classList coverage for IE9, which could be handled another way. It doesn't make sense for things with near universal native support like dom lookups and event handling. Just a warning from the grave of someone who migrated code out of jQuery 👻